### PR TITLE
Explicit Relative Imports

### LINKED ISSUE #40: Use explicit relative imports within CiviWiki modules
We should use **explicit relative imports** within a module. This promotes portability and readability.

# Goal
Change CiviWiki modules to use relative imports where possible.

# Task
The following files/lines can be changed to use relative imports.

## /api/models/account.py
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/models/account.py#L20-L22

## /api/models/activity.py
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/models/activity.py#L2-L4

## /api/models/civi.py
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/models/civi.py#L2-L5
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/models/civi.py#L12

## /api/models/notification.py
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/models/notification.py#L2-L4

## /api/models/rationale.py
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/models/rationale.py#L2-L4

## /api/models/rebuttal.py
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/models/rebuttal.py#L2-L3

## /api/models/response.py
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/models/response.py#L2-L3

## /api/models/thread.py
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/models/thread.py#L2-L6

## /api/models/vote.py
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/models/vote.py#L2-L3

## /api/write.py
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/write.py#L11
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/write.py#L14
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/dev/api/write.py#L23

## /api/consumers.py
- [ ] https://github.com/CiviWiki/OpenCiviWiki/blob/dbacc71c188135cdd4344aca09813ebb1b70b8a2/api/consumers.py#L4

### PR DESCRIPTION
**Pull request by pscheid2**
Closes #40

First commits. Never coded in an open source project so let me know if I'm not following any guidelines. 

The dev environment setup was pretty self explanatory. The one improvement I think we could make is to add a little section about setting up PostgreSQL (creating a user and a initial DB), so I might do that.

Hope I did the explicit relative imports right...

I'll merge this branch into dev after its looked at

### CODE REVIEW

File: project/api/write.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 from api.forms import UpdateProfileImage
 from django.core.files import File   # need this for image file handling
 from django.core.files.base import ContentFile
-from api.models import Thread
+from api.models.thread import Thread
```
Comment (brylie) [inline project/api/write.py]: As far as I can tell, `.models.thread` should work here, since they are in the same module namespace.

**Comment by pscheid2** [comment]
added the requested changes

**Comment by brylie** [comment]
@jl24 any suggestions for this PR? If not, we can merge.

**Comment by brylie** [comment]
@jl24 any feedback, or should this be merged?

**Pull request merged**